### DISCUSSION TITLE
Welcome to PEGS_Mixture Discussions!

### DISCUSSION THREAD

**Discussion by kyle-messier** [kyle-messier/PEGS_Mixture#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kyle-messier** [comment]
@sigmafelix  starting some discussion here.

BART and other bayesian tree methods could be a viable approach for the mixtures analysis. Here are a few other things I cam across through the PRIME website and other searches:

1) https://pubmed.ncbi.nlm.nih.gov/34562017/  dlmtree (has an R package) distributed lag models better estimate mixture interaction effects 
2) https://mural.maynoothuniversity.ie/17359/1/thesis__Alan_.pdf This thesis describes some novel visuzalitions for tree methods such as BART - could help with mixture interpretability

**Comment by sigmafelix** [comment]
Thank you for the suggestions. I think the tree-based methods are good for their interpretability and their being in machine learning methods. I will look into the papers.

**Comment by kyle-messier** [comment]
@sigmafelix A lot of studies reported at AGU are using Childs et al 2022 study that maps wildfire specific pm2.5

**Comment by sigmafelix** [comment]
@Spatiotemporal-Exposures-and-Toxicology Thank you for sharing the article. For the record, the study [link](https://doi.org/10.1021/acs.est.2c02934) is here.
